schema/schema.yaml

```
@@ -1,48 +1,24 @@
 $schema: https://fiboa.github.io/schema/v0.1.0/schema.json
 required:
-  - ml:creation_method
   - ml:boundary_type
+  - ml:creation_date
   - ml:verified_from
-  - ml:verified_until
   - ml:author
-  - ml:quality
-  - ml:machine_generated
   - ml:country_code
-  - ml:admin0
   - ml:admin1
   - ml:admin2
-  - ml:crop_category
 properties:
-  ml:creation_method:
-    type: string
-  ml:creation_method_detail:
-    type: string
   ml:ml:boundary_type:
     type: string
-    required: true
   ml:creation_date:
     type: date-time
   ml:verified_from:
     type: date-time
-  ml:verified_until:
-    type: date-time
   ml:author:
     type: string
-  ml:quality:
-    type: string
-  ml:confidence:
-    type: float
-  ml:machine_generated:
-    type: boolean
   ml:country_code:
     type: string
```

```suggestion
    type: string
    minLength: 3
    maxLength: 3
    pattern: ^[A-Z]{3}$
```

To actually validate whether the the country code are somewhat valid.